src/cmd/model_main.cc

```diff
@@ -146,8 +146,9 @@ void addModelFlags(CLI::App &_app)
     ->default_val("");
 
   command->add_flag("-p,--pose", opt->pose,
-                "Print the pose of the model.")
-                ->needs(modelCmd);
+    "Print the pose of the model. (only useful with --model)");
+  // ->needs(modelCmd) -- CLI11 < 2.5.0 does not support this
```

Comment:
I commented in https://github.com/gazebosim/gz-sim/issues/2918#issuecomment-2898490755 that `->needs(modelCmd)` also works properly with 1.9.1. It seems like a regression between 1.9.1 and 2.4.2 that was fixed in 2.5.0

Reply:
ok! reverted in [a967ca7](https://github.com/gazebosim/gz-sim/pull/2910/commits/a967ca78a23a7eec6269a9ebe98a97c939d2694f)